ts/features/bonus/cgn/screens/merchants/CgnMerchantsCategoriesSelectionScreen.tsx

```diff
@@ -87,7 +90,9 @@ const CgnMerchantsCategoriesSelectionScreen = () => {
 
   const categoriesToArray: ReadonlyArray<ProductCategoryEnum | "All"> = [
     "All",
-    ...(pot.isSome(potCategories) ? potCategories.value : [])
+    ...(pot.isSome(potCategories)
+      ? orderCategoriesByNameKey(potCategories.value)
```

Comment:
Just a proposal for a minor optimization, what you think to move `orderCategoriesByNameKey` from here to the reducer:
```typescript
    case getType(cgnCategories.success):
      return {
        ...state,
        list: pot.some(orderCategoriesByNameKey(action.payload))
      };
``` 

In this case the list will be ordered only when a new data is received and not also when we move from `pot.some -> pot.someLoading` (it's already ordered in that case).

Reply:
good point! [9402a42](https://github.com/pagopa/io-app/pull/3765/commits/9402a42fbc8c53d3a0c2b72881cf836e17b12a7c) implements it